### ISSUE TITLE
SSH Key path

### ISSUE THREAD

**Issue by Gino** [LaravelCollective/remote#48]
Hi,

I am wondering if there is an option to specify the ssh key itself instead of the path.
Since it is pretty weird that I have to specify a file path instead of the ssh key itself.

Thanks.

**Comment by stidges** [comment]
You can pass in a key directly by using the `keytext` configuration option instead of the `key` option.

**Comment by dotZoki** [comment]
This could be mentioned in documentation with maybe more description of remote.php configuration keyes.

**Comment by chrisx84** [comment]
i agree, im having a hell of a time trying to use my ssh key in this package.

**Comment by ryanhalley** [comment]
If using windows, and a key file, remember to also use forwards slashes e.g: 'd:/private_key.txt'
